custom-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -0,0 +1,325 @@
+            },
+            "GroupType": {
+              "type": "string",
+              "description": "The group type.",
+              "title": "Type"
+            },
+            "Email": {
+              "type": "string",
+              "description": "The groupmail.",
+              "title": "Email"
+            },
+            "Owners": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "The emails of all the owners.",
+                "title": "Owners emails"
+              },
+              "description": "Owners"
+            },
+            "Members": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "The emails of all the members.",
+                "title": "Members emails"
+              },
+              "description": "Members"
+            },
+            "isPublic": {
+              "type": "boolean",
+              "description": "If the group is public.",
+              "title": "Is public",
+              "enum": [
+                "",
+                "true",
+                "false"
+              ]
+            },
+            "CreateTeam": {
+              "type": "boolean",
+              "description": "If the group should create a Team.",
+              "title": "Create a Team",
+              "enum": [
+                "",
+                "true",
+                "false"
+              ]
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "summary": "When a group is created",
+        "description": "This will trigger when a new group is created within GroupMgr.",
+        "operationId": "GroupMgrGroupTrigger",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "description": "Name for this approval flow",
+                  "title": "Approval name",
+                  "x-ms-visibility": "important"
+                },
+                "webhook": {
+                  "type": "string",
+                  "description": "The webhook URL",
+                  "title": "Webhook",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                }
+              },
+              "required": [
+                "name",
+                "webhook"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/api/removeWebhook/{id}": {
+      "delete": {
+        "responses": {
+          "default": {
```

Comment:
@Jwaegebaert I would recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.